tribute_page.html

```diff
@@ -1,20 +1,22 @@
-<body id='main'>
-  <header>
-    <section id="title">
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Microverse Trials Review Code</title>
+    <link rel="stylesheet" href="style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  </head>
+  <body>
       <h1>Dr. Norman Borlaug</h1>
       <h2 id="tribute-info">The man who saved a billion lives</h2>
       <div id="img-div">
-        <img id = "image" class="center" class="responsive" src="https://www.pbs.org/wgbh/americanexperience/media/filer_public_thumbnails/filer_public/56/7c/567c2141-1572-4455-9078-3ddc8609e127/borlaug-article-1.jpg__1000x567_q85_crop_subsampling-2.jpg" id="image"/ alt="Dr. Norman Borlaug training biologists in Mexico">
-          <a href="https://www.pbs.org/wgbh/americanexperience/features/green-revolution-norman-borlaug-race-to-fight-global-hunger/" target="_blank"></a>
-        </img>
-      <div>
-       <p for="image" id="image-label"><em>
-        Borlaug smiling in wheat field. Image, 1960s. International Maize and Wheat Improvement Center (CIMMYT)</em></p></div>
```

Comment:
We know sometimes is hard to find a good image and we have to use from other sources/sites. Remember to identify from who is the source and grant the images credits.